script/Ciel/Day 12/12_01_CIEL12_3B.txt

```diff
@@ -307,22 +339,24 @@ I steal a glance at Ciel-senpai.
 [sha:b11ede70be08e4c2b71c4af33393b7346ad8911f]{
 -- Page 115, Offset 31163.
 -- この人は街で派手に遊ぶより、公園をのんびり散策したりする方が性に合っているような気がして、どうも誘えない。
-I can't ask her out. After all, she seems more suited to a leisurely stroll at the park than she is playing around in the city.
+I feel like she'd be way more at home taking a leisurely stroll around the park than having an afternoon out on the town.
+ There's no way I could propose something like that to her.
 }
 [sha:e1515589d1e6cc22eb47fd98484aaf02281b4bef]{
 -- Page 116, Offset 31164.
 -- これから街に遊びに行こうと言っても、先輩は断ってくると思うし―――
-Even if I were to invite her, I'm sure she would refuse―――
+Even if I did invite her, I'm sure she'd refu―――
 }
 [sha:d16553a5070c20f740601d0bf228505fc30a6f71]{
 -- Page 117, Offset 31165.
 -- 「ふむ。乾くんの意見はもっともです。
-"Hmm. I agree with you, Inui-kun.
+"Hmm. I think that's an excellent idea, Inui-kun.
 }
 [sha:e9c8528fbb7ec5e336c234d6f906e1ffd9d8c035]{
 -- Page 117, Offset 31166.
 -- 遠野くんへのご褒美もありますし、せっかくなので三人で遊びに行きましょうか」
-Not to mention that I have to reward Tohno-kun, and since we're here, how about the three of us go out and have some fun?"
+I owe Tohno-kun a reward anyway,
+ so why don't we take advantage of this opportunity to go have some fun together?"
 }
 [sha:b20547082dd04b2a6c8a2dd41c0c62e321c94a00]{
 -- Page 117, Offset 31167.
```

Comment:
31168 seems weird

>Really Senpai?

Or something is probably more natural. Then again you don't use 'Senpai' in English so lol.